edb/tools/test/__init__.py

```diff
@@ -154,6 +154,31 @@ def test(*, files, jobs, include, exclude, verbose, quiet, debug,
     sys.exit(result)
 
 
+def _prepare_coverage_result_folder(container_folder: pathlib.Path) -> None:
```

Comment:
I think we can still use a temporary directory for intermediate files, but write the final `.coverage` file the the current directory.  That way you don't need this code.  Dealing with other generated files (such as HTML) would be left to the user.

Reply:
Thanks! I didn't know that .coverage output file is sufficient to generate reports (I understand now it aggregates all results). I'll do it :+1: